packages/tooltips/src/styled/StyledTooltip.ts

```diff
@@ -13,23 +13,17 @@ import {
   DEFAULT_THEME,
   getLineHeight
 } from '@zendeskgarden/react-theming';
-
-import { POPPER_PLACEMENT, getArrowPosition } from '../utils/gardenPlacements';
+import { getArrowPosition } from '../utils/gardenPlacements';
+import { ITooltipProps, POPPER_PLACEMENT } from '../utils/types';
 import { StyledParagraph } from './StyledParagraph';
 import { StyledTitle } from './StyledTitle';
 
 const COMPONENT_ID = 'tooltip.tooltip';
 
-export type TOOLTIP_SIZE = 'small' | 'medium' | 'large' | 'extra-large';
-export type TOOLTIP_TYPE = 'light' | 'dark';
-
-export interface IStyledTooltipProps {
-  hasArrow?: boolean;
-  size?: TOOLTIP_SIZE;
+export interface IStyledTooltipProps extends Pick<ITooltipProps, 'hasArrow' | 'size' | 'zIndex'> {
   /** All valid [Popper.JS Placements](https://popper.js.org/popper-documentation.html#Popper.placements) */
-  placement: POPPER_PLACEMENT;
-  zIndex?: number | string;
-  type: TOOLTIP_TYPE;
+  placement: typeof POPPER_PLACEMENT[number];
+  type: NonNullable<ITooltipProps['type']>;
 }
```

Comment:
Should we move `styled/` types under the same umbrella? Or is it strictly for the public API?
If so, maybe a `types/styled.ts` would be a good way of organizing it without necessarily exporting it.

Reply:
I'm opposed to re-situating these private interfaces away from their associated styled components to a central location. I feel like it communicates the wrong thing re: visibility – but that's just my personal preference and I'm happy to defer to the team's direction on this. To your point, we might discover some nice type affinity that could be gained if these are co-located 🤷.

Either way, my current tooling exercise is not going to engage with the `styled` interfaces. That's a substantial effort; my plan is to stay focused on the publicly exported element component interfaces.